esp-radio/src/ble/npl.rs

```diff
@@ -1186,6 +1196,37 @@ pub(crate) fn ble_init() {
                     msys_cnt2: u16,
                     from_heap: u8,
                 ) -> i32;
+
+                fn base_stack_initEnv() -> i32;
+                fn conn_stack_initEnv() -> i32;
+                fn adv_stack_initEnv() -> i32;
+                fn extAdv_stack_initEnv() -> i32;
+                fn sync_stack_initEnv() -> i32;
+            }
+
+            let res = base_stack_initEnv();
+            if res != 0 {
+                panic!("base_stack_initEnv returned {}", res);
```

Comment:
Should we make `ble_init` fallible? The amount of panics (which should be asserts, or bubbled-up errors) is way too much